python/xt/dune/xt/common/vtk/plot.py

```diff
@@ -168,3 +168,73 @@ def plot(vtkfile_path, color_attribute_name, interactive=False, color_map=DEFAUL
             IPython.display.display(hbox)
 
         return vtkplot
+
+    class NumpyPlot(k3dPlot):
+
+        def __init__(self,
+                     vertices,
+                     indices,
+                     data,
+                     color_attribute_name='Data',
+                     color_map=k3d.basic_color_maps.CoolWarm,
+                     interactive=False,
+                     *args,
+                     **kwargs):
+            super().__init__(*args, **kwargs)
+
+            self.idx = 0
+
+            if 'transform' in kwargs.keys():
+                raise RuntimeError('supplying transforms is currently not supported for time series VTK Data')
+
+            self.mesh = k3d.mesh(vertices=vertices,
+                                 indices=indices,
+                                 color=0x0000FF,
+                                 opacity=1.0,
+                                 attribute=data,
+                                 color_range=(np.nanmin(data), np.nanmax(data)),
+                                 color_map=np.array(color_map, np.float32),
+                                 wireframe=True,
+                                 compression_level=0)
+            self += self.mesh
+
+            self.camera_no_pan = not interactive
+            self.camera_no_rotate = not interactive
+            self.camera_no_zoom = not interactive
+
+    def grid_plot(gridprovider, color_attribute_name, interactive=False, color_map=DEFAULT_COLOR_MAP):
```

Comment:
The `color_attribute_name` is note required/used any more, is it?

Reply:
Yeah. You're right it's only used in the VTK stuff to select the right XML attribute.